### PR TITLE
LRQA-54492

### PR DESCRIPTION
**Pull request by nkkwon**
https://issues.liferay.com/browse/LRQA-54492
AppBuilder#UnpublishAndRepublish

Needs https://github.com/alexcandidos/liferay-portal/pull/101

### CODE REVIEW

**Comment by nkkwon** [comment]
ci:test:sf

**Comment by nkkwon** [comment]
ci:test:relevant

File: modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase [hunk 1]
```diff
@@ -1579,6 +1579,186 @@ definition {
 		AppBuilderAdmin.validateIfSearchItemIsNotFound(itemName = "Apple");
 	}
 
+	@description = "Unpublish and Republish an App"
+	@priority = "5"
+	test UnpublishAndRepublish {
+		property portal.acceptance = "false";
```
Comment (alexcandidos) [inline modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase]: Hey @nkkwon , Thank you for the improvements! Could you please just remove the annotation related to the acceptance test? Thanks!

**Comment by alexcandidos** [comment]
ci:test:app-builder

**Comment by alexcandidos** [comment]
ci:test:app-builder

**Comment by alexcandidos** [comment]
ci:test:relevant

**Comment by alexcandidos** [comment]
Pull request forwarded to https://github.com/brianchandotcom/liferay-portal/pull/84087. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by alexcandidos** [comment]
Hey @nkkwon , could you please take a look on the conflicts ? Thanks!

**Comment by nkkwon** [comment]
Fixed all the conflicts @alexcandidos

**Comment by alexcandidos** [comment]
Pull request forwarded to https://github.com/brianchandotcom/liferay-portal/pull/84258. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by alexcandidos** [comment]
Thanks @nkkwon !

**Pull request closed without merging**